### PR TITLE
Addition of bl_info

### PR DESCRIPTION
**Pull request by romio82**
Hi Nathan,
this is Romuald. We talked over at the Rhino forum..
I'm very excited about your 3DM Importer - so much so that I decided to follow your invitation and set my first insecure steps into the world of branches and forks.
As I wanted to avoid making embarrassing mistakes right away(which I'm probably still doing somewhere) I searched for a tiny task for my fist pull request so that I could get a glimpse of how things work in here: So here is a first shining and blinking version of a bl_info directly out of my fresh fork :) If you have some time, please let me know what you think of it..
Speaking of "please let me know"": If you don't want to have any bloody beginners like me in your repo, please tell me - I'd absolutely understand and would further on follow your magical achievements with the importer from farther away without any bad feelings. But if you unexpectedly had some small tasks that you would like to hand over and if you had the time to do some boring reviews, I would be more than happy to try my best! I've already learned a lot and that's quite cool!
Please tell me frankly if I could be of any use here..
Thank you for your efforts!
Best, Romuald

### CODE REVIEW

File: import_3dm.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
+bl_info = {
+    "name": "Import Rhinoceros 3D",
+    "author": "jesterKing",
```
Comment (jesterKing) [inline import_3dm.py]: You can add your own name as well:

```suggestion
    "author": "jesterKing, romio82",
```

File: import_3dm.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+    "version": (0, 0, 1),
+    "blender": (2, 80, 0),
+    "location": "File > Import > Rhinoceros 3D (.3dm)",
+    "description": "This Addon lets you import all render meshes from Rhinoceros 3dm files",
```
Comment (jesterKing) [inline import_3dm.py]: I would word this a little bit different

```suggestion
    "description": "This addon lets you import Rhinoceros 3dm files",
```

File: import_3dm.py [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+    "warning": "",
```
Comment (jesterKing) [inline import_3dm.py]: We can tell here that not all data is yet supported

```suggestion
    "warning": "The importer doesn't handle all data in 3dm files yet",
```

File: import_3dm.py (hunk 2 above)
Comment (romio82) [reply import_3dm.py]: I had same text as you suggest in the beginning but the I thought there should be information on the fact that for the time being only rendermeshes are supported. This is of course solved in a much clearer way in the "warning" line as you suggested! :) I applied these changes.

File: import_3dm.py (hunk 1 above)
Comment (romio82) [reply import_3dm.py]: I'd rather add my name as soon as I contribute something substantial to the code.. ;) I'll have some time over the weekend. Maybe I'll get some small addition ready that would be worth mentioning my name..

**Pull request merged**